.github/workflows/smoketest.yml

```diff
@@ -0,0 +1,76 @@
+name: bridge
+
+on:
+  push:
+    paths:
+      - "bridges/snowbridge/**"
+      - "!bridges/snowbridge/README.md"
+      - "!bridges/snowbridge/LICENSE"
+    branches:
+      - snowbridge
+  workflow_dispatch:
+
+jobs:
+  smoketests:
+    runs-on: snowbridge-runner
+    env:
+      CARGO_INCREMENTAL: 0
+      RUST_BACKTRACE: 1
+      RUSTFLAGS: -C debuginfo=1
+    outputs:
+      success: ${{ steps.run_e2e_stack.outcome == 'success' }}
+    steps:
+      - name: Set build directory environment variable
+        run: echo "OUTPUT_DIR=${GITHUB_WORKSPACE}/tmp" >> $GITHUB_ENV
+      - name: Set log directory environment variable
+        run: echo "LOG_DIR=${GITHUB_WORKSPACE}/log-tmp" >> $GITHUB_ENV
+      - name: Create directories
+        run: mkdir -p $OUTPUT_DIR &&  mkdir -p $LOG_DIR
+      - uses: actions/checkout@v2
+        with:
+          repository: snowfork/snowbridge
+          ref: smoketests-ci
```

Comment:
The issue is that contracts are in the Snowbridge repo and the rust code is in this repo. So the rust code in the current branch is paired together with the contracts from Snowbridge repo using this `ref` which contains the branch. This makes it like having a submodule. Maybe we should re-introduce submodule?

Reply:
Whoops, I switched to branch `smoketests-ci` to test the workflow, but it should actually be `main`. I updated it in https://github.com/Snowfork/polkadot-sdk/pull/113/commits/28f5c47649169d2a52c1b0f8652b778151bf5825. It is sort of arranged like a submodule, but not actually using a submodule means we don't have to update the submodule link every time we make a change to the polkadot-sdk.

This workflow will only run on changes made to the `snowbridge` branch, not on every code commit. So the onlu assumption here is that the `main` branch should already be updated when merging PRs in this repo (which companion changes in snowfork/snowbridge).